src/Core/Resolvers/SqlPaginationUtil.cs

```diff
@@ -115,6 +115,34 @@ public static JsonDocument CreatePaginationConnectionFromJsonDocument(JsonDocume
             return result;
         }
 
+        /// <summary>
+        /// Holds the information safe to expose in the response's pagination cursor,
+        /// the NextLink. These can then be used to form the pagination columns that
+        /// will be needed for the actual query.
+        /// </summary>
+        protected class NextLinkColumn
+        {
+            public string EntityName { get; set; }
+            public string FieldName { get; set; }
+            public object? Value { get; }
+            public string? ParamName { get; set; }
+            public OrderBy Direction { get; set; }
+
+            public NextLinkColumn(
+                string entityName,
+                string fieldName,
+                object? value,
+                string? paramName = null,
+                OrderBy direction = OrderBy.ASC)
+            {
+                EntityName = entityName;
+                FieldName = fieldName;
+                Value = value;
```

Comment:
is there a better name for this? either more descriptive or a comment describing what 'value' represents.

Reply:
i changed it to FieldValue, but this is just the value of the column, a database row or record being a set of columns.